src/components/SignupForm/SignupForm.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import { Button, Form, Input, Radio, Upload } from 'antd';
+import './signup-form.less';
+
+const SignupForm: React.FC = () => {
+  const onFinish = (values: any) => {
```

Comment:
This is the one thing im gonna nitpick, we want to make best use of Typescript here and attempt to get this validated! As you can see heres an example of values:
![image](https://user-images.githubusercontent.com/23691775/97195728-0e6ace00-1782-11eb-88d3-efd7053bd942.png)

Can we make this a typescript interface with all these fields and use that type instead of any? Really minor thing but imo its important :)